v1/ao/internal/reporter/reporter_grpc.go

```diff
@@ -781,6 +781,7 @@ func (r *grpcReporter) sendMetrics(msg []byte) {
 	case errInvalidServiceKey:
```

Comment:
Both this and above "Invalid key error cases", seems need logging an error message